sections/experiments/results.tex

```diff
@@ -19,7 +19,45 @@ \subsubsection{LDOS-CoMoDa}
 \\\\
 To summarize, we expected the context-aware methods to perform well on the LDOS-CoMoDA dataset, as the ratings are context-dependent. 
-The \texttt{CAMF} methods do perform well as expected, but \texttt{Itemsplitting} is underperforming in most metrics.
+The \texttt{CAMF} methods do perform well as expected, but \texttt{IS-UserBased} is underperforming in most metrics.
 Some baselines such as \texttt{SVD} and \texttt{SVD++} are closely competitive with \texttt{CAMF} as the best performing methods.
 The results of the experiment could also indicate that \texttt{NGCF} could be interesting to look at in terms of adding context, as it is able to perform well on a context dataset without utilizing the context available. 
 \texttt{NGCF} is also a graph-based approach which, as mentioned in \autoref{sec:graph-rec-sys}, is able to alleviate the sparsity problem introduced by adding context.
+
+\subsubsection{MovieLens 100k}
+The MovieLens 100k dataset does not contain any contextual variables outside of a timestamp that can be discretized.
+\autoref{subsec:experimentprotocol} defined how the timestamp was discretized into the day of the week and a time interval.
+Compared to the LDOS-CoMoDa we expected context-aware methods to not perform substantially better since only context based on the timestamp is available compared to the different contextual variables included in the LDOS-CoMoDa dataset. 
+In terms of the methods that handle context, we see that CAMF-C achieves the highest precision and F1, while IS-UserBased underperforms most methods on precision@N and outperforms all methods on recall@N.
+In terms of NDCG IS-UserBased underperforms on MovieLens compared to other datasets, showing the worst results, while both CAMF methods also provide poor NDCG.
+KGAT, a graph-based method utilizing side information performs the best in terms of NDCG on the MovieLens data, but underperforms on MAP@10, precision and F1.
+Other graph-based methods, such as NGCF and LightGCN show similar poor performances as well on this dataset, being outperformed on precision@10, recall@10, F1 and MAP@10.
+DeepWalk excels on the MAP@10 metric and outperforms other graph based methods on precision, recall and F1, showing results close to SVD-based approaches.
+In terms of RMSE and MAE, we see that both CAMF and both SVD approaches perform the best, with random underperforming as expected.
+Overall, we see a tendency for the graph-based methods to perform poorly on this dataset except for DeepWalk, while the SVD-based approaches perform the best in terms of F1.
+
+\subsubsection{Yahoo Movies}
+For the Yahoo Movies dataset we opted to use side information such as \textit{gender} and \textit{age group} as the contextual variables for the context-aware method since the dataset does not contain any contextual information with the recorded interactions.
```

Comment:
```suggestion
For the Yahoo Movies dataset, we opted to use side information such as \textit{gender} and \textit{age group} as the contextual variables for the context-aware method since the dataset does not contain any contextual information with the recorded interactions.
```